protecode/scanning_util.py

```diff
@@ -402,10 +402,15 @@ def enumerate_reference_triages():
         # yield results
         for protecode_app in protecode_apps_to_consider:
             scan_result = self._api.scan_result(protecode_app.product_id())
+
+            def pdf_report_callback():
```

Comment:
naming-remark: this is not actually a callback, is it?

a brief remark as to _why_ we pass the "promise"-like callable might be a good idea (avoid loading everything into memory longer than necessary)